### DISCUSSION TITLE
cpp cublas

### DISCUSSION THREAD

**Discussion by Policing2024** [SubtitleEdit/subtitleedit#8382] [General]
No text found
when transcribing with
whisper/cpp/model/large-q5_0.bin
Attached is the err msg pls.

![CPP CUBLAS.txt](https://github.com/SubtitleEdit/subtitleedit/files/15348747/CPP.CUBLAS.txt)

**Comment by niksedk** [comment]
Not really an SE issue, try in the Whisper forums

**Comment by Policing2024** [comment]
I look for the forum, but i did'nt get it pls

**Comment by Policing2024** [comment]
Sir
What of Models with asteric "*" pls

**Reply by niksedk** [reply]
`*`in the download combobox shows that the model is already downloaded:

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/7e2f9129-008e-4a6a-958d-36b125263434)

**Comment by Policing2024** [comment]
NO TEXT FOUND ERR WITH:
large-v3-fp16
large-v3-int8

err msg attached pls

On Fri, May 24, 2024, 18:00 Nikolaj Olsson ***@***.***> wrote:

> *in the download combobox shows that the model is already downloaded:
>
> image.png (view on web)
> <https://github.com/SubtitleEdit/subtitleedit/assets/1261331/7e2f9129-008e-4a6a-958d-36b125263434>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SubtitleEdit/subtitleedit/discussions/8382#discussioncomment-9549905>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BAA5ONQAXFGLICOABHP6CADZD5W2TAVCNFSM6AAAAABH377HYOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TKNBZHEYDK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>

Date: 05/31/2024 01:57:44
SE: 4.0.5.349 - Microsoft Windows NT 10.0.22635.0 - 64-bit
Message: D:\SE 405\Whisper\Purfview-Whisper-Faster\whisper-faster.exe --language hi --model "large-v3-fp16" --task translate  "D:\AA\RAKSHAK 1996 DVDRIP XVID UNTOUCH AC3 Striker WBZ_2024_05_27_22_09_42_00001.mp4"

-----------------------------------------------------------------------------
Date: 05/31/2024 01:58:23
SE: 4.0.5.349 - Microsoft Windows NT 10.0.22635.0 - 64-bit
Message: Calling whisper (Purfview's Faster-Whisper) with : D:\SE 405\Whisper\Purfview-Whisper-Faster\whisper-faster.exe --language hi --model "large-v3-fp16" --task translate  "D:\AA\RAKSHAK 1996 DVDRIP XVID UNTOUCH AC3 Striker WBZ_2024_05_27_22_09_42_00001.mp4"
Warning: Word-level timestamps on translations may not be reliable.

Warning: 'large-v3' model may produce inferior results, try 'large-v2'!

Standalone Faster-Whisper-XXL r192.3.4 running on: CPU
Starting work on: D:\AA\RAKSHAK 1996 DVDRIP XVID UNTOUCH AC3 Striker WBZ_2024_05_27_22_09_42_00001.mp4
ValueError: Invalid input features shape: expected an input with shape (1, 128, 3000), but got an input with shape (1, 80, 3000) instead

File "faster_whisper\transcribe.py", line 1109, in encode

File "faster_whisper\transcribe.py", line 798, in generate_segments

File "faster_whisper\transcribe.py", line 1456, in restore_speech_timestamps

File "D:\whisper-fast\_XXL\__main__.py", line 1595, in cli

File "D:\whisper-fast\_XXL\__main__.py", line 1668, in <module>

Traceback (most recent call last):

[9744] Failed to execute script '__main__' due to unhandled exception!
Calling whisper Purfview's Faster-Whisper done in 00:00:38.2809593
Loading result from STDOUT


-----------------------------------------------------------------------------
Date: 05/31/2024 02:00:13
SE: 4.0.5.349 - Microsoft Windows NT 10.0.22635.0 - 64-bit
Message: D:\SE 405\Whisper\Purfview-Whisper-Faster\whisper-faster.exe --language hi --model "large-v3-int8" --task translate  "D:\AA\RAKSHAK 1996 DVDRIP XVID UNTOUCH AC3 Striker WBZ_2024_05_27_22_09_42_00001.mp4"

-----------------------------------------------------------------------------
Date: 05/31/2024 02:00:51
SE: 4.0.5.349 - Microsoft Windows NT 10.0.22635.0 - 64-bit
Message: Calling whisper (Purfview's Faster-Whisper) with : D:\SE 405\Whisper\Purfview-Whisper-Faster\whisper-faster.exe --language hi --model "large-v3-int8" --task translate  "D:\AA\RAKSHAK 1996 DVDRIP XVID UNTOUCH AC3 Striker WBZ_2024_05_27_22_09_42_00001.mp4"
Warning: Word-level timestamps on translations may not be reliable.

Warning: 'large-v3' model may produce inferior results, try 'large-v2'!

Standalone Faster-Whisper-XXL r192.3.4 running on: CPU
Starting work on: D:\AA\RAKSHAK 1996 DVDRIP XVID UNTOUCH AC3 Striker WBZ_2024_05_27_22_09_42_00001.mp4
ValueError: Invalid input features shape: expected an input with shape (1, 128, 3000), but got an input with shape (1, 80, 3000) instead

File "faster_whisper\transcribe.py", line 1109, in encode

File "faster_whisper\transcribe.py", line 798, in generate_segments

File "faster_whisper\transcribe.py", line 1456, in restore_speech_timestamps

File "D:\whisper-fast\_XXL\__main__.py", line 1595, in cli

File "D:\whisper-fast\_XXL\__main__.py", line 1668, in <module>

Traceback (most recent call last):

[1788] Failed to execute script '__main__' due to unhandled exception!
Calling whisper Purfview's Faster-Whisper done in 00:00:36.7535193
Loading result from STDOUT

**Comment by Policing2024** [comment]
No text found err, above pls
